### ISSUE TITLE
Doesn't work with rollup@0.48

### ISSUE THREAD

**Issue by Vlad-Zhukov** [rollup/rollup-plugin-commonjs#220]

**Comment by frzi** [comment]
Error I get:
```
TypeError: Cannot read property 'slice' of undefined
    at startsWith (/project/node_modules/rollup-plugin-commonjs/dist/rollup-plugin-commonjs.cjs.js:612:12)
    at resolveId (/project/node_modules/rollup-plugin-commonjs/dist/rollup-plugin-commonjs.cjs.js:653:23)
    at Object.options (/project/node_modules/rollup-plugin-commonjs/dist/rollup-plugin-commonjs.cjs.js:711:27)
    at /project/node_modules/rollup/dist/rollup.js:9738:41
    at Array.reduce (<anonymous>)
    at new Bundle$$1 (/project/node_modules/rollup/dist/rollup.js:9737:25)
    at Object.rollup (/project/node_modules/rollup/dist/rollup.js:10532:16)
    at Object.<anonymous> (/project/rollup.tasks.js:107:10)
    at Module._compile (module.js:573:30)
    at Object.Module._extensions..js (module.js:584:10)
```

**Comment by cedx** [comment]
On my side: this issue occurs only if I use `input` instead of `entry` in the Rollup config file. If I keep the old `entry` property, I get a warning ("options.entry is deprecated, use options.input") but the build succeeds.

**Comment by Rich-Harris** [comment]
Fixed in 8.1.1 — thanks!
